### ISSUE TITLE
accelerate the acceptance test

### ISSUE THREAD

**Issue by qibobo** [redhat-developer/service-binding-operator#778]
When I create a new PR or do some change for a PR, it takes more than 1 and a half hours to run the acceptance test. 
So is there a way to reduce the duration? Maybe is it possible to run the acceptance tests in parallel?

**Comment by DhritiShikhar** [comment]
+1

**Comment by pedjak** [comment]
Thanks for submitting the issue. Are you referring CI runs on those running locally?

**Comment by qibobo** [comment]
@pedjak 
Both in travis or in local machine are slow. So it would be better if it can run parallel.

**Comment by pedjak** [comment]
By merging https://github.com/redhat-developer/service-binding-operator/pull/880 CI jobs takes ~20mins only to complete.

Also, by specifying `EXTRA_BEHAVE_ARGS` on the command line, one can execute just scenario of a given name, for example:

```
make test-acceptance EXTRA_BEHAVE_ARGS="-n 'My Scenario'"
```
